.github/build/friendly-filenames.json

```diff
@@ -0,0 +1,33 @@
+{
+    "android-arm64": { "friendlyName": "android-arm64-v8a" },
+    "darwin-amd64": { "friendlyName": "macos-64" },
+    "darwin-arm64": { "friendlyName": "macos-arm64-v8a" },
+    "dragonfly-amd64": { "friendlyName": "dragonfly-64" },
+    "freebsd-386": { "friendlyName": "freebsd-32" },
+    "freebsd-amd64": { "friendlyName": "freebsd-64" },
+    "freebsd-arm64": { "friendlyName": "freebsd-arm64-v8a" },
+    "freebsd-arm7": { "friendlyName": "freebsd-arm32-v7a" },
+    "linux-386": { "friendlyName": "linux-32" },
+    "linux-amd64": { "friendlyName": "linux-64" },
```

Comment:
this should be `linux-amd64` in order to not change the format from previous release